nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/computation/ComputationContext.java

```diff
@@ -66,4 +66,10 @@ default void produceRecord(String streamName, String key, byte[] data) {
      * @since 10.1
      */
     void askForTermination();
+
+    /**
+     * @return the offset of the last record read.
+     * @since 10.3
+     */
+    long getLastOffset();
```

Comment:
You should return a LogOffset that contains the partition and log name in case of error you need all this information

Reply:
Definitly, will do !